packages/kotti-ui/source/kotti-table/standard-table/hooks.ts

```diff
@@ -48,9 +58,23 @@ export const useKottiStandardTable = <ROW extends AnyRow>(
 } => {
 	const params = computed(() => paramsSchema.parse(_params.value))
 
+	const appliedFilters = ref<KottiStandardTable.AppliedFilter[]>([])
+	/**
+	 * https://github.com/TanStack/table/discussions/4670
+	 */
+	const globalFilter = ref<KottiFieldText.Value>(' ')
```

Comment:
Apparently Column filters aren't applied until the Global filter is applied. Maybe this was fixed on a more recent version of tanstack table